src/rp/index.ts

```diff
@@ -165,7 +168,18 @@ export class RelyingPartySolution {
       })
       .toSign();
 
-    return vp_token;
+    const [, vc_scope] = requestObj.payload.scope;
+    const presentation_submission = {
+      definition_id: "32f54163-7166-48f1-93d8-ff217bdb0653",
```

Comment:
According to https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#response-parameters I believe this may be a description of the type of presentation we are doing. For example `pid_presentation`

Reply:
As per the provided example in https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/relying-party-solution.html, I think we can use the first part of the scope field in the request object. Such a field I think is in the format `definition_id vc1 vc2 ... vcN` (not clear in specs).

Done in https://github.com/pagopa/io-react-native-wallet/pull/22/commits/c2ef778ec4a7ac83f893cf2b50a77ec62911db05